helix-term/src/ui/picker.rs

```diff
@@ -159,16 +159,32 @@ impl<T: 'static> Component for FilePicker<T> {
         // |picker   | |         |
         // |         | |         |
         // +---------+ +---------+
-        let render_preview = area.width > MIN_SCREEN_WIDTH_FOR_PREVIEW;
-        let area = inner_rect(area);
+        let area = inner_rect(
+            area,
+            cx.editor.config.file_picker.width_factor,
+            cx.editor.config.file_picker.height_factor,
+        );
+        let render_preview = area.width
+            > cx.editor.config.file_picker.preview_min_width
+                + cx.editor.config.file_picker.file_min_width;
```

Comment:
This is a bit repetitive, add `let config = &cx.editor.config.file_picker` at the top of `render`